R/promptAndRun.R

```diff
@@ -36,6 +42,7 @@ promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
         next
       } else {
         coupled <- c(coupled, paste0(paste0(myruns[[i]], "/output/", runnames[[i]], "-rem-"), seq(10)))
+        coupled <- c(coupled, paste0(myruns[[i]], "/output/", runnames[[i]])) # for coupled runs in parallel mode
```

Comment:
This is a preparation for https://github.com/remindmodel/remind/pull/794, because if the runs are run in parallel mode, the runname is `C_SSP2-Base-rem-1` or so. As all non-existing are filtered out anyway, it shouldn't bother.

Reply:
ok